.github/workflows/ci.yaml

```diff
@@ -37,7 +37,7 @@ jobs:
           pip install .[tests]
 
       - name: Check out MS tool
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
```

Comment:
I updated a lot of actions here, but nothing interesting - just node 20 upgrades.